### ISSUE TITLE
Question: Optimised Size

### ISSUE THREAD

**Issue by nevi-me** [web-dom/web-dom#1]
Just saw this project on crates.io, looks great!

In the alert demo, the wasm file loaded is over 300KB. Have any optimisations been applied to it? A concern with wasm is often the binary size, so I think adding some info on how much size the generated files from both examples (console::log() and alert()) end up being, might be useful.

**Comment by richardanaya** [comment]
Good idea, there's various Rust tools out there I know exist, and some general guidelines.  I'm a bit unfamiliar with the latest and greatest ways to optimize wasm.  It would be nice to have a section "Here's how to get the smallest wasm possible".
